Issue #304 - enable ctrl + click

SanSunB opened the pull request:
Issue #304 - enable ctrl + click

Enabled ctrl+click  on <a> and navigation function in search results cards.
Added cursor - pointer to the area around the result link text.

akariv commented:
Hey @SanSunB  - a few comments:
First, sorry it tool such a long time for me to get to this.
Second - 
the pull request goes 'against the browser' and not with it :) this means that we want the browser to provide the link opening functionality, and not re-implement it in code.
This means that if we want to support ctrl+click on an element, the best thing would be to wrap it with an `<a ..>` element with the proper href instead of implementing a complex `click` handler.
(also - not that adding `target="_blank"` to an `a` element will mean that _all_ links will open in a new tab, which was not the intended result.

SanSunB commented:
There was already click event on the search result which cause the ctrl+click not to work when pressing on the original <a> element. Since i implemented the <a> on all the section the click event had, and they don't differ in the functionality, i deleted the click event and the href() function is called from the <a> only. I left the navigate() in case it is used from a different place.
